tests/backends/tests.py

```diff
@@ -745,6 +745,19 @@ def test_cursor_contextmanager_closing(self):
             self.assertIsInstance(cursor, CursorWrapper)
         self.assertTrue(cursor.closed)
 
+    @skipUnlessDBFeature('test_db_allows_multiple_connections')
```

Comment:
Regarding testing, I was suggesting:

* add the test to all three backends specifically (making sure to use TransactionTestCase since that is where the bug presented)
* in addition, add a test that applies to all backends that calling the method doesn't raise an exception